tests/v3_api/common.py

```diff
@@ -452,6 +452,22 @@ def validate_cluster(client, cluster, intermediate_state="provisioning",
     return cluster
 
 
+def validate_dns_record(pod, record, expected):
+    #requires pod with `dig` available (sangeetha/testclient)
+    host = '{0}.{1}.svc.cluster.local'.format(
+        record["name"], record["namespaceId"])
+
+    cmd = 'ping -c 1 -W 1 {0}'.format(host)
+    output = kubectl_pod_exec(pod, cmd)
+    assert "0% packet loss" in str(output)
```

Comment:
@bmdepesa You should be able check additionally that ping response has one of the entries from expected